.drone.yml

```diff
@@ -0,0 +1,51 @@
+pipeline:
+   node 9:
+      image: node:9.10.1
```

Comment:
Use just `node:9` please so we don't have to update this for each release of node

Reply:
Changed to node:9